src/GeneralStateTestsFiller/stTransactionTest/CreateTransactionEOF1Filler.yml

```diff
@@ -30,8 +30,21 @@ CreateTransactionEOF1:
       #- ':yul { mstore8(0, 0xef) return(0, 32) }'
       - ':label deploying_0xef00..00 :raw 0x60ef60005360206000f3'
 
+      # EIP 3670: Valid (legacy) init code containing truncated PUSH data at the end
+      # { mstore(0x00, 0xef00010100060060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push1(0x1, }
```

Comment:
Cosmetic: I think all these (legacy init code) cases should go into section `# Legacy initcode => EOF code` below.